flickit-assessment-core/src/main/java/org/flickit/assessment/core/adapter/in/rest/confidencelevel/CalculateConfidenceLevelResponseDto.java

```diff
@@ -0,0 +1,6 @@
+package org.flickit.assessment.core.adapter.in.rest.confidencelevel;
+
+import org.flickit.assessment.core.application.port.in.confidencelevel.CalculateConfidenceLevelUseCase.Result;
+
+public record CalculateConfidenceLevelResponseDto(Result confidenceLevel) {
```

Comment:
rename record to `CalculateConfidenceResponseDto`
rename parameter to `confidenceValue`